cmd/internal/collector/collector.go

```diff
@@ -97,7 +98,13 @@ func (cc CCloudCollector) handleResponse(response QueryResponse, ccmetric CCloud
 		}
 
 		if topicPresent && clusterPresent && rule.ShouldIgnoreResultForRule(topic, cluster, ccmetric.metric.Name) {
-			continue
+			continue METRICSLOOP
```

Comment:
By curiosity, why replacing `continue` with `continue METRICSLOOP`? :)

Reply:
Yeah, they are interchangeable in this case, I just did it since the label needed in the next loop and wanted to keep it consistent? but I'll remove the change in the next commit.